src/Provider/AwsProvider.php

```diff
@@ -311,13 +312,15 @@ public function queueExists()
             return true;
         }
 
-        $result = $this->sqs->getQueueUrl([
-            'QueueName' => $this->getNameWithPrefix()
-        ]);
+        try {
+            $result = $this->sqs->getQueueUrl([
+                'QueueName' => $this->getNameWithPrefix()
+            ]);
 
-        if($this->queueUrl = $result->get('QueueUrl')) {
-            return true;
-        }
+            if ($this->queueUrl = $result->get('QueueUrl')) {
+                return true;
```

Comment:
Do you mind setting the queueUrl back into cache here?  Its an attempt to keep us from having to hit the API to keep track of it, any time we do, let's reset the cache.